backend/src/ledgers/data.ts

```diff
@@ -31,42 +33,31 @@ const REDIS_PAGING_TOKEN_KEY = {
   month: "paging_token_month",
 };
 
-const HORIZON_LIMITS = {
-  hour: 10,
-  day: 100,
-  month: 200,
+export const REDIS_LEDGER_KEYS = {
+  hour: "ledgers_hour",
+  day: "ledgers_day",
+  month: "ledgers_month",
 };
 
-const intervalTypes = [INTERVALS.day, INTERVALS.hour, INTERVALS.month];
-
-const BIGQUERY_DATES = {
-  hour: getBqDate(INTERVALS.hour),
-  day: getBqDate(INTERVALS.day),
-  month: getBqDate(INTERVALS.month),
+export const REDIS_AVGS_LEDGER_KEYS = {
```

Comment:
doesn't look like `REDIS_AVGS_LEDGER_KEYS` is used. Are they meant to be? Otherwise let's remove